src/xrGame/WeaponMagazined.cpp

```diff
@@ -90,7 +90,7 @@ void CWeaponMagazined::Load(LPCSTR section)
     m_sSndShotCurrent = "sndShot";
 
     //звуки и партиклы глушителя, если такой есть
-    if (m_eSilencerStatus == ALife::eAddonAttachable || m_eSilencerStatus == ALife::eAddonPermanent)
+    if ((m_eSilencerStatus == ALife::eAddonAttachable || m_eSilencerStatus == ALife::eAddonPermanent) && WeaponSoundExist(section, "snd_silncer_shot"))
```

Comment:
Please, don't disable flame particles when shot sound doesn't exist. This code does exactly that.